aict-MERN/aict-site-w-react/src/components/Main/Main.js

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react'
+import './main.css'
+import MakeAPost from './components/MakeAPost/MakeAPost'
+import Posts from './components/Posts/Posts'
+import CreatePostBtn from './components/createPostBtn/CreatePostBtn'
+
+
+const Main = () => {
+    const [makeAPostVisible, setMakeAPostVisible] = useState('none')
+
+    return(
+        <>
+            <main>
+                {/* <section className="forum-heading">
+                    <div className="create-post-header">
+                        <h2 className="create-post-header">Create a Post!</h2>
+                        <p>
+                            Our forum is a place for any coder or programmer to ask questions, make comments, really anything you want. 
+                        </p>
+                    </div>
+                </section> */}
+                <section className="posts">
```

Comment:
i didnt know section was a valid html attribute! :o

Reply:
They are all basically the same lol but yeah I think it is a perfect descriptive tag name

Reply:
its neato! :D